### PR TITLE
Pixel: update letter spacing and line height

### LINKED ISSUE #6455: Pixel: Styles: Typography — Letter Spacing & Line Height
Update following typography settings.

* Letter spacing
* Line height

**For**
* Headings
* Paragraphs
* Site-title
* Post-title

Design: Cm51iCQSPu8Ar1qufDTeOQ-fi

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

Updates Pixel's line height and letter spacing values. 

#### Related issue(s):

Closes #6455

### CODE REVIEW

File: pixel/theme.json [hunk 1]
```diff
@@ -364,32 +363,44 @@
 			},
             "h1": {
                 "typography": {
-                    "fontSize": "3rem"
+                    "fontSize": "3rem",
+                    "letterSpacing": "0.05em",
+                    "lineHeight": "1.2"
```
Comment (madhusudhand) [inline pixel/theme.json:379]: set `letterSpacing` and `lineHeight` in `heading` element and override the lineHeight in `h6`

**Review by madhusudhand — COMMENTED** [review]
update the lineHeight of site-title to 1.2

File: pixel/theme.json (hunk 1 above)
Comment (jffng) [reply pixel/theme.json:379]: Done in https://github.com/Automattic/themes/pull/6487/commits/231bd9319f1583ac7606ea2ab5b47071be1fd6ff.

**Review by madhusudhand — APPROVED** [review]
LGTM!

**Pull request merged**